package.json

```diff
@@ -97,9 +95,13 @@
     "@angular-devkit/core": "~7.3.3",
     "@angular/compiler-cli": "~7.2.6",
     "@angular/language-service": "~7.2.6",
+    "@types/express": "^4.16.1",
     "@types/jasmine": "~3.3.9",
     "@types/jasminewd2": "~2.0.6",
     "@types/mocha": "~5.2.6",
+    "@types/node": "^11.10.5",
```

Comment:
not 100% sure but i believe there was an issue with idam that taleb and russell experienced when they used the latest version of this. I also experienced it but i set my version to 8.0.0 which fixed the idam. not sure if its the same issue with them.

Reply:
now it's in docker I believe it uses node 11, I will see if Azure can give us that too